pkg/apis/kops/validation/aws.go

```diff
@@ -79,3 +83,24 @@ func awsValidateMachineType(fieldPath *field.Path, machineType string) field.Err
 
 	return allErrs
 }
+
+// TODO: make image validation smarter? graduate from jessie to stretch? This is quick and dirty because we keep getting reports
+func awsValidateAMIforNVMe(fieldPath *field.Path, ig *kops.InstanceGroup) field.ErrorList {
+	// TODO: how can we put this list somewhere better?
+	NVMe_INSTANCE_PREFIXES := []string{"P3", "C5", "M5", "H1", "I3"}
```

Comment:
@geojaz is it possible that this array is missing the r5 family? https://aws.amazon.com/ec2/instance-types/r5/